src/sentry/tagstore/snuba/backend.py

```diff
@@ -16,69 +17,363 @@
 from sentry.tagstore.base import TagStorage
 from sentry.utils import snuba
 
+SEEN_COLUMN = 'received'
+
+
+class ObjectWrapper(object):
+    def __init__(self, dictionary):
+        self.__dict__ = dictionary
+
 
 class SnubaTagStorage(TagStorage):
 
+    def get_time_range(self, days=90):
+        """
+        Returns the default (start, end) time range for querrying snuba.
+        """
+        end = datetime.utcnow().replace(tzinfo=pytz.UTC)
+        return (end - timedelta(days=days), end)
+
     # Tag keys and values
     def get_tag_key(self, project_id, environment_id, key, status=TagKeyStatus.VISIBLE):
-        pass
+        # TODO is status applicable?
+        return self.get_group_tag_key(project_id, None, environment_id, key)
```

Comment:
I didn't look up where we depend on this again but we may need to wrap this to catch `GroupTagKeyNotFound` and turn it into `TagKeyNotFound`. Same with other method "re-uses."